.github/workflows/run_tests.yml

```diff
@@ -9,7 +9,7 @@ on:
       boards_to_test:
         type: string
         required: false
-        default: '["nrf52840dk_nrf52840", "nrf5340dk_nrf5340_cpuapp", "nrf54l15pdk_nrf54l15_cpuapp"]'
+        default: '["nrf52840dk/nrf52840", "nrf5340dk/nrf5340/cpuapp", "nrf54l15pdk/nrf54l15/cpuapp"]'
```

Comment:
Based on GH Actions build - it doesn't work for nRF54l

Reply:
this looks like another issue unrelated to this line change.

Reply:
I choose this line because it's related with gh action, and the issue is present there.
The issue is only with the new hardware model, not on mainline with the normal one